test/loss/test_loss_wrapper.py

```diff
@@ -32,6 +32,50 @@ def test_masking(self):
         # print((grad[~mask] == 0).sum())
         self.assertTrue((grad[~mask] == 0).all())
 
+    def test_ApplyMask_output_shape_crop(self):
```

Comment:
Can you add a test where the number of channels in `p` and `t` is > 1, but the mask only has a single channel?

Reply:
done